src/System.Console/tests/packages.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Collections" version="4.0.10-beta-22412" />
+  <package id="System.IO" version="4.0.10-beta-22412" />
+  <package id="System.IO.FileSystem" version="4.0.0-beta-22412" />
+  <package id="System.Runtime" version="4.0.20-beta-22412" />
+  <package id="System.Runtime.Extensions" version="4.0.10-beta-22412" />
+  <package id="System.Threading" version="4.0.0-beta-22412" />
+  <package id="System.Threading.Overlapped" version="4.0.0-beta-22412" />
+  <package id="System.Threading.Tasks" version="4.0.10-beta-22412" />
+  <package id="System.Threading.Thread" version="4.0.0-beta-22412" />
+  <package id="System.Threading.ThreadPool" version="4.0.10-beta-22412" />
```

Comment:
Are all these dependencies actually necessary? I didn't see any reference to ThreadPool for example.